docs/gettsim_objects/variables_out.rst

```diff
@@ -42,3 +42,5 @@ documentation of all :ref:`functions`.
 +-------------------------+-----------------------------------------------------------+
 | wohngeld_m_hh           | Monthly housing benefit per household                     |
 +-------------------------+-----------------------------------------------------------+
+| grunds_im_alter_m_hh    | Monthly subsistence payment for retirees per household    |
```

Comment:
As far as I understand, the Regelbedarf, as well as the considered income is always considered on the level of the household (Bedarfsgemeinschaft). Do you think it is different? And is it different to ALG 2 (because that one is similarly only calculated on the household level).

A problematic case is of course if one person in the Bedarfsgemeinschaft is retired and the other is not. In these cases GETTSIM currently makes the following simplification:

```
    # ToDo: Grundsicherung im Alter is only paid if all adults in the household
    # ToDo: are retired. Other households get ALG 2 (This is a simplification by
    # ToDo: GETTSIM)

Reply:
you are right, sorry. I got confused between grundrente and grundsicherung im Alter here.